build-image.sh

```diff
@@ -10,7 +10,6 @@ shared_lib="$(dirname $0)/.shared"
 [ -e "$shared_lib" ] || curl -sSf https://raw.githubusercontent.com/vegardit/docker-shared/v1/download.sh?_=$(date +%s) | bash -s v1 "$shared_lib" || exit 1
 source "$shared_lib/lib/build-image-init.sh"
 
-
```

Comment:
Can you please leave the two lines. This visual separation of logical blocks is a personal preference.